packages/react/src/components/TimePicker/TimePickerDropdown.jsx

```diff
@@ -215,13 +245,8 @@ const TimePickerDropdown = ({
       e.relatedTarget?.parentNode?.classList[0].includes('iot--list-spinner');
 
     if (!contained) {
-      // close dropdown and validate
+      // close dropdown
       setOpenState(false);
-      setInvalidState(!validate(inputRef.current.value));
```

Comment:
The removal of this code now allows for values other than numbers to be entered into the input.

Reply:
Go to [this](https://next.carbon-addons-iot-react.com/?path=/story/1-watson-iot-timepickerdropdown--default) story and try typing in letters. With your changes the [same story](https://deploy-preview-3523--carbon-addons-iot-react.netlify.app/?path=/story/1-watson-iot-timepickerdropdown--default) now allows for these inputs to go through.